unified-installer/astra-unified-installer.sh

```diff
@@ -1117,12 +1179,177 @@ is_astra_registry() {
     local -r registry_url="$1"
     if [ -z "$registry_url" ]; then fatal "no registry_url given"; fi
 
-    if echo "$registry" | grep -q "cr." && echo "$registry" | grep -q "astra"; then
+    if echo "$registry_url" | grep -q "cr." && echo "$registry_url" | grep -q "astra"; then
```

Comment:
Bug; variable got renamed at some point but these references weren't changed alongside it.